### DISCUSSION TITLE
It says there are "Outstanding issues" with my account and won't let me apply for a students account

### DISCUSSION THREAD

**Discussion by Andrew-71** [community/community#19209] [GitHub Education]
*I asked this [on community forum](https://github.community/t/it-says-there-are-outstanding-issues-with-my-account-when-applying-for-a-students-account/257998) already but was kindly asked to post here instead.*

I finally decided to apply for GitHub students account today. *However*, when I tried sending my application I was immediately greeted with this message:

>There are outstanding issues with your GitHub account. Please contact [GitHub Support](https://support.github.com/contact) for more information.

I think I already know what it may be caused by: I live in Moscow, however I usually spend part of the year in Crimea for health reasons, and so my account was falsely tagged as a sanctioned one some time ago. I never bothered to fix that, since I didn't want sending a picture of my passport to GitHub for privacy reasons (and it doesn't affect much except for no private repositories).

I still believe it’s a good idea to ask here thought in case it’s some other issue. It would be extremely disappointing if the sanctioned account status is the reason for the deny, as I believe it is unfair to discriminate students based on their location, while calling the program open “to students from all countries”.

I want to state that I have provided a valid school email that matches my real life school, located in Moscow, and has allowed the website access my location to prove that I am currently in the city as well. I triple checked all requirements and don't see any other reason for me to be automatically rejected immediately.

Edit: I have finally, after a long time, bothered to make an appeal on status of my account, so hopefully I will at least get the Student status once the appeal is reviewed.

**Comment by melvincwng** [comment]
Hi @Andrew-71, for account-related issues, please kindly raise a ticket to [Github Support](https://support.github.com/contact/account).

Hope this clarifies.

**Comment by RabieTAWIL** [comment]
Dear @Andrew-71 ,
I hope you are doing well. I recently wanted also to participate in the Student Developer Pack, and got the same error that you had. Similarly, I think my problem is similar to yours where my account were once accessed from a region that is under some restriction, and I didn't do anything about it that time, but I wish I did.

I would like to ask if you solved your problem, and if you can help and guide me to solve mine in case you solved yours.

Thank you in advance.
